src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -74,12 +76,11 @@
        :icon     :i/delete
        :id       :delete-for-me}])
    (when (cond
-           deleted?        false
-           deleted-for-me? false
```

Comment:
So, users can delete already deleted-for-me message?

Reply:
Yes, that was discussed with @John-44 , I think that can be helpful